src/YouTube.js

```diff
@@ -121,7 +121,21 @@ class YouTube extends React.Component {
    * @param {Object} event
    *   @param {Object} target - player object
    */
-  onPlayerReady = (event) => this.props.onReady(event);
+  onPlayerReady = (event) => {
+    if (!this.props.iframeAttrs) {
+      this.props.onReady(event);
+      return;
+    }
+
+    this.internalPlayer
+      .getIframe()
```

Comment:
I would have liked to make this synchronous, so the timing of `onPlayerReady` -> `this.props.onReady` wouldn't have an extra tick added in... but `event.target` doesn't seem to have a stable public name for the iframe. In the linked issue repro it's `event.target.a`, but in my local testing it was `event.target.h`. Thoughts?